app/src/main/java/be/digitalia/mediasession2mqtt/MainWorker.kt

```diff
@@ -110,25 +107,30 @@ class MainWorker @Inject constructor(
             val device: DeviceInfo
         )
 
-        // TODO: using a data class for storing sensor metadata would be nicer and let us configure
-        //  icons etc. but this suffices for a proof-of-concept.
-        val sensors = mapOf(
-            "Application ID" to "$APPLICATION_ID_SUB_TOPIC",
-            "Playback state" to "$PLAYBACK_STATE_SUB_TOPIC",
-            "Media title" to "$MEDIA_TITLE_SUB_TOPIC"
+
+        val deviceInfo = DeviceInfo(
+            name = "MediaSession2MQTT",
+            manufacturer = Build.MANUFACTURER,
+            model = Build.MODEL,
+            identifiers = listOf(deviceId.toString())
         )
 
-        val deviceInfo = DeviceInfo()
+
+        val sensors = listOf(
+            Pair("Application ID", "$APPLICATION_ID_SUB_TOPIC"),
```

Comment:
using the notation `x to y` in Kotlin is a shortcut to create a `Pair(x, y)`. My remark was just to change `mapOf()` with `listOf()` and keeping the rest unchanged.

Reply:
Understood & changed.